### ISSUE TITLE
Backend 403s

### ISSUE THREAD

**Issue by albertyw** [vinitkumar/currently#10]
It seems currently's backend call to https://s3.amazonaws.com/currently-notifications/notifications.production.json is now failing with a 403, along with the parent website http://rainfalldesign.com/ which got taken over by a domain squatter.  Did you ever get in contact with the currently owners?

**Comment by vinitkumar** [comment]
@albertyw Nope, I never had any touch with the currently owners. Looks like the company went out of business or something.
